index.html

```diff
@@ -5,9 +5,28 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" href="style.css"/>
-        <title>title</title>
+        <title>Барбершоп «Бородинский»</title>
     </head>
     <body>
+        <header>
+            <nav>
+                <a>
+                <img src="svg/index-logo.svg" width="368" height="120" alt="Логотип барбершопа «Бородинский»">
+                </a>
+                <ul>
+                    <li> <a href="#">Информация</a> </li>
+                    <li> <a href="#">Новости</a> </li>
+                    <li> <a href="#">Прайс-лист</a> </li>
+                    <li> <a href="#">Магазин</a> </li>
+                    <li> <a href="#">Контакты</a> </li>
```

Comment:
```suggestion
                    <li> 
                        <a href="#">Информация</a>
                    </li>
```

и так будет с каждым, вложенность, у тебя на лишке будут классы потом, будет не читабельно